courses/models_test.py

```diff
@@ -114,6 +137,20 @@ def test_program_page_properties():
         assert block.value == "benefit"
     assert not LearningOutcomesPage.can_create_at(program_page)
 
+    # ForTeamsPage tests
```

Comment:
as we are adding more, we need to decouple these tests for each subtype.

Reply:
Done.